openfecli/commands/quickrun.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import click
+from openfecli import OFECommandPlugin
+import json
+
+
+@click.command(
+    'quickrun',
+    short_help="Run a given transformation, saved as a JSON file"
+)
+@click.argument('transformation', type=click.File(mode='r'),
+                required=True)
+def quickrun(transformation):
+    """Run the transformation (edge) in the given JSON file in serial.
+
+    To save a transformation as JSON, use the following Python recipe (after
+    creating the transformation):
+
+    \b
+        import json
+        with open(filename, 'w') as f:
+            json.dump(transformation.to_dict(), f)
+    """
+    import gufe
+    from gufe.protocols.protocoldag import execute
+    print("Loading file...")
+    dct = json.load(transformation)
```

Comment:
also worth thinking about wrapping each step in a try/except and catching any errors?  (speaking of, what are these errors...)

Reply:
If there are errors that we can handle, then we could. If this is just to avoid dumping a stack trace, then let click do it for it (it does it automatically).